src/components/tabController/TabPage.tsx

```diff
@@ -79,25 +83,23 @@ export default function TabPage({
   [currentPage, lazyLoad]);
 
   const animatedPageStyle = useAnimatedStyle(() => {
-    const isActive = Math.round(currentPage.value) === index;
-
-    // TODO: Fix to proper animated style once Reanimated export AnimatedStyleProp
-    const style: any = {
-      opacity: isActive || asCarousel ? 1 : 0,
-      zIndex: isActive || asCarousel ? 1 : 0
-    };
-
-    if (nestedInScrollView) {
-      style.position = isActive ? 'relative' : 'absolute';
+    if (!nestedInScrollView) {
+      return {};
     }
 
-    return style;
+    const isActive = Math.round(currentPage.value) === index;
+
+    return {position: isActive ? 'relative' : 'absolute'};
   });
 
   const _style = useMemo(() => {
     return [!asCarousel && styles.page, animatedPageStyle, {width: asCarousel ? containerWidth : undefined}, style];
   }, [asCarousel, animatedPageStyle, containerWidth, style]);
 
+  if (!isActive && !asCarousel) {
```

Comment:
By returning null you cancel the TabController behavior of keeping pages "alive" (but hidden)
So each tab change will remount the tab page children 
In member view for example, it will load all the page sections and will generally degrade the performance

Reply:
Changed to `opacity` + `zIndex` change in a regular style, seems to work (although it is above the RN warning\error - not sure how it was before).
Need to do some further tests, so change to a draft